src/OedipusRex.jl

```diff
@@ -27,3 +26,184 @@ function ReX(v=20,w=20)
     # Oh yikes, and it explodes as u->0 takes it to ->0 in the denominator
 end
 
+"""
+NOTE: This is old code that has been replaced by the full χ function.
```

Comment:
We should either uncomment or delete this whole function.

Reply:
Yeah, I agree. I have just included it for my own benefit for now as I suspect I will actually still use this code for evaluating the chi function at small temperatures as I recall it converging faster than the integrals.